### ISSUE TITLE
Benchmarking support (query times)

### ISSUE THREAD

**Issue by vlucas** [vlucas/Spot#22]
It would be nice to have a setting for turning on query benchmarking/timing so query execution time can be reported with some sort of method - maybe in `Spot\Log`. This would allow optimization of slow or long-running queries.

**Comment by marvelade** [comment]
you mean like in cakePHP?

**Comment by vlucas** [comment]
Yes - and like the profiler bar in Symfony and other applications. This would just be adding support to GET the MEASURE the query times from Spot though, so this wouldn't be the whole profiler bar or anything.

**Comment by brandonlamb** [comment]
I think i will look at a solution this week. Getting an array with query plus time would be cool to output how you like
